<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,20,0,0">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnLoadFmData">
         <property name="text">
          <string>load fm data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSim">
         <property name="text">
          <string>Similarity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGFD">
         <property name="text">
          <string>TEST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGen">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalculate">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="graphletId">
         <property name="text">
          <string>graphlet:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="nodeNumEdit">
         <property name="placeholderText">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edgeNumEdit">
         <property name="placeholderText">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbxDistMeasure">
         <item>
          <property name="text">
           <string>L1 Distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L2 Distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cosine</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalcMeanDist">
         <property name="text">
          <string>calculate mean distance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalcMeanTime">
         <property name="text">
          <string>calculate mean time</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxRwFlag">
      <item>
       <property name="text">
        <string>random walk</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>enumerate</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxGFDCalc">
      <item>
       <property name="text">
        <string>Accumulate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Concatenate</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxKernel">
      <item>
       <property name="text">
        <string>Cosine Similarity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RBF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laplacian</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxRange"/>
    </item>
    <item>
     <widget class="QWidget" name="container" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Widget" name="m_widget_0" native="true"/>
       </item>
       <item>
        <widget class="Widget" name="m_widget_1" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fileListBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnClearFileList">
      <property name="text">
       <string>Clear File list</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header location="global">widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
